src/syl.cpp

```diff
@@ -124,6 +124,9 @@ int Syl::PrepareLyrics(FunctorParams *functorParams)
     }
 
     this->SetStart(dynamic_cast<LayerElement *>(this->GetFirstAncestor(NOTE, MAX_NOTE_DEPTH)));
+    // If there isn't an ancestor note, it should be a chord
+    if (!this->GetStart())
+        this->SetStart(dynamic_cast<LayerElement *>(this->GetFirstAncestor(CHORD, MAX_CHORD_DEPTH)));
```

Comment:
This line does not use a code block for a single statement.  i.e., wrapping in `{}`.  Is there a style guideline for that in verovio?  (the other added code is all using code blocks for single statements, so that is why I am pointing it out).  I personally always use code block because that minimizes the chances of hidden bugs, and it also makes it easier to add code to a single-statement block in the future.

Reply:
@craigsapp  See https://github.com/rism-ch/verovio/blob/master/doc/guidelines.md#exception-for-one-line-control-clauses
The only exception remains with only if statements (without else) that can be written on a single line.

Reply:
added them anyway

Reply:
>  Is there a style guideline for that in verovio?

Yes. Basically WebKit coding guidelines, with one exception precisely regarding single-line statements. They should **always** be put in a block:
https://github.com/rism-ch/verovio/blob/master/doc/guidelines.md#exception-for-one-line-control-clauses

Reply:
> The only exception remains with only if statements (without else) that can be written on a single line.

Correct. However, when formatting put them on several lines, it is better to add a block.